docs/experimental-features/security-posture-management.asciidoc

```diff
@@ -0,0 +1,46 @@
+[[security-posture-management]]
+== Cloud security posture management
+
+NOTE: This feature requires Elastic Stack version 8.3.0 or later.
+
+Cloud security posture management (CSPM) and Kubernetes security posture management (KSPM) continuously monitor and compare your cloud and kubernetes infrastructure against security best practices in order to help you identify and remediate misconfigurations. 
+
+[discrete]
+=== Enable security posture management
+
+The steps to enable this feature differ between self-hosted and Elastic cloud deployments.
+
+[[enable-kspm-on-cloud]]
+[discrete]
+==== Enable KSPM for cloud deployments
+
+First, enable the KSPM flag for your deployment:
+
+* From the left-side navigation menu in Kibana, click on **Manage this deployment**.
+
+image::images/kspm-1.png[The Manage deployment button]
+
+* Under "My deployment", select **Kibana**.
+* Under "Instances", click the three-dot menu next to your instance, then click **Edit configuration**.
+* Click **Edit user settings** in the upper right of the page.
+* Add the following line to the User settings: `xpack.cloudSecurityPosture.enabled: true`.
+* Next, click the **Back** button under User settings, and then click **Save** in the lower left corner, and confirm your changes.
```

Comment:
```suggestion
* Click **Back** then save your Kibana user settings.
```

Reply:
Here are the Kibana instructions if you want to model these similarly: https://www.elastic.co/guide/en/cloud/current/ec-manage-kibana-settings.html#ec-manage-kibana-settings